src/components/EventDashboard/CCPTabPanel.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  IconButton,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableSortLabel,
+  TableRow,
+} from '@material-ui/core';
+import { Add, MoreHoriz } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+import { useQuery } from '@apollo/react-hooks';
+import { Colours } from '../../styles/Constants';
+import { Order, stableSort, getComparator } from '../../utils/sort';
+import { GET_CCPS_BY_EVENT_ID } from '../../graphql/queries/ccps';
+
+const useStyles = makeStyles({
+  root: {
+    padding: '56px 56px 145px 56px',
+  },
+  tableContainer: {
+    background: Colours.White,
+    border: `1px solid ${Colours.BorderLightGray}`,
+    borderRadius: '4px',
+  },
+  addButton: {
+    boxShadow: '0px 4px 10px rgba(0, 0, 0, 0.3)',
+    borderRadius: '2000px',
+    position: 'fixed',
+    bottom: '56px',
+    right: '56px',
+    padding: '12px 26px',
+  },
+  visuallyHidden: {
+    border: 0,
+    clip: 'rect(0 0 0 0)',
+    height: 1,
+    margin: -1,
+    overflow: 'hidden',
+    padding: 0,
+    position: 'absolute',
+    top: 20,
+    width: 1,
+  },
+  buttonIcon: {
+    marginRight: '13px',
+  },
+});
+
+interface CCP {
+  id: string;
+  name: string;
+  eventId: { id: string };
+}
```

Comment:
can this be pulled out an reused elsewhere? this entity is generic/ubiquitous enough that it shouldn't live in this component (or maybe it should if we really don't use this anywhere else)